### ISSUE TITLE
LogEntry-level tags support strings only

### ISSUE THREAD

**Issue by rr-media** [getsentry/sentry-dotnet#260]
Hello, guys.

Currently, only string values will be added to event tags if you use `_logger.Log` with format arguments. So if I log a message with string parameter, the entry will be sent with additional **stringTag**

`
                    this.logger.LogError("Some Application error: {stringTag}", "value");
`

But the similiar message with int parameter will be sent without additional tag

`
                    this.logger.LogError("Some Application error: {intTag}", 100500);
`

It looks a bit inconsistent imo.
Is this any reason behind this? Probably we can support not only strings but other primitive types / DateTime / Guid? 

Thanks

**Comment by bruno-garcia** [comment]
@rr-media Is this about `Sentry.Extensions.Logging`? Or are you using Serilog or else?
We should handle values other than string too. Sounds like something we should take a look.

**Comment by rr-media** [comment]
Yeah, it's about `SentryLogger` in `Sentry.Extensions.Logging` package. 
It's not clear if these non-string values should go to 'Tags' collection or probably in 'Extra', but it will be definitely helpful to have them in the event.

**Comment by bruno-garcia** [comment]
We send the whole formatted string:

https://github.com/getsentry/sentry-dotnet/blob/12c8df6a412963d65935dcacf0ef0f4defa48711/src/Sentry.Extensions.Logging/SentryLogger.cs#L70-L80

And the also the template (so Sentry can group on the log line and not create an issue for each formatted message).
I'd say this is an enhancement then to send the values in Extra or perhaps even better, in a new section just for the log "name, value" for structured logging.
